src/Libraries/CoreNodes/FileSystem.cs

```diff
@@ -76,6 +76,7 @@ public static string ReadText(FileInfo file)
         /// <param name="path"></param>
         /// <param name="newPath"></param>
         /// <param name="overwrite"></param>
+        /// <returns name="void">Node performs a task, doesn’t produce an output </returns>
```

Comment:
I think it's strange to add a `returns` tag when a method does not return anything. Does it actually do anything?

Also, I don't see descriptions for the parameters, which is actually important.

Reply:
this was added to give an explanation into what void means
![image](https://user-images.githubusercontent.com/25138291/106265228-a2838000-621e-11eb-97b5-826e24fef4b3.png)

Reply:
Hi @mmisol, I think I adressed all of the comments on existing Prs, the only thing left to resolve is this

> this was added to give an explanation into what void means
> ![image](https://user-images.githubusercontent.com/25138291/106265228-a2838000-621e-11eb-97b5-826e24fef4b3.png)

Reply:
Thanks @martinstacey . I was in doubt this had any effect but it seems we are actually using it. Please ignore the first part of my comment.

Please add descriptions for `path`, `newPath` and `overwrite` right above.

Reply:
Addressed in 5e21593

Reply:
> Thanks @martinstacey . I was in doubt this had any effect but it seems we are actually using it. Please ignore the first part of my comment.
> 
> Please add descriptions for `path`, `newPath` and `overwrite` right above.

Addressed in 5e21593